0-SCore/Scripts/UtilityAI/UAITaskAttackTargetEntitySDX.cs

```diff
@@ -196,7 +196,14 @@ public override void Update(Context _context)
                     _context.Self.Attack(true);
                     break;
                 case 1:
-                    if (!_context.Self.Use(false)) return;
+                    // Use() doesn't trigger the weapon fire animation, do that here.
+                    // Trigger it now so the first call to Use() has the correct animation.
+                    _context.Self.emodel.avatarController.TriggerEvent("WeaponFire");
+                    if (!_context.Self.Use(false))

```

Comment:
I honestly don't know how `Use` could return false, but I figured that if it did, we probably want to cancel the "WeaponFire" event triggered on the controller. Maybe that's not needed, but I think better safe than sorry.